contrib/pyln-proto/examples/connect.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+"""Simple connect and read test
+
+Connects to a peer, performs handshake and then just prints all the messages
+it gets.
+
+"""
+
+from pyln.proto.wire import connect, PrivateKey, PublicKey
+from binascii import unhexlify, hexlify
+
+ls_privkey = PrivateKey(unhexlify(
+    b'1111111111111111111111111111111111111111111111111111111111111111'
+))
+remote_pubkey = PublicKey(unhexlify(
+    b'03b31e5bbf2cdbe115b485a2b480e70a1ef3951a0dc6df4b1232e0e56f3dce18d6'
+))
+
+lc = connect(ls_privkey, remote_pubkey, '127.0.0.1', 19375)
```

Comment:
`9735` ? Or is it intended ?

Reply:
I was using a non-standard port for testing. Will change.